assets/css/style.css

```diff
@@ -1,276 +1,398 @@
+  display: flex;
+  align-content: center;
+  padding-left: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
   color: #7a869a;
-  font-size: 1.4rem;
+  column-gap: 2em;
+  list-style: initial;
 }
 
-.card-body p {
-  margin: 0.5rem 1rem 0 0;
-  font-family: var(--font-family-style);
-  font-style: normal;
+.ul-details > li {
+  list-style: disc;
+}
+
+.p-ul-details {
+  font-size: 15px;
   font-weight: 400;
-  font-size: 13px;
   line-height: 24px;
-  color: var(--color-paragraphs);
+  color: #344563;
 }
 
-.categories {
+.ul-tags {
   display: flex;
-  align-items: center;
-  gap: 0.8rem;
   list-style: none;
-  margin-top: 1rem;
+  column-gap: 4px;
+  margin: 15px 0 15px 0;
 }
 
-.categories li {
-  font-size: 1.1rem;
-  background: #ebebff;
-  color: var(--accent-color);
-  padding: 0.5rem 1rem;
+.ul-tags li {
+  padding: 3px 14px 3px 14px;
+  background-color: #ebebff;
   border-radius: 8px;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  font-weight: 500;
 }
 
-.btn {
-  padding: 1.5rem 2.2rem;
-  margin-top: 3rem;
-  border-radius: 10px;
-  border: 1px solid #396df2;
+.btn-in {
+  cursor: pointer;
+  background-color: var(--white);
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  margin: 15px 0 15px 0;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  line-height: 24px;
   color: #396df2;
+  font-weight: 500;
+  transition: 0.7s;
 }
 
-.btn:hover {
+.btn-in:hover {
   background-color: #6070ff;
   color: var(--white);
 }
 
-.btn:active {
+.btn-in:active {
   background-color: #2230d2;
   color: var(--white);
 }
 
-.about-page {
-  background: var(--white);
-  border-top-right-radius: 75px;
-  justify-content: center;
-  align-items: flex-start;
-  max-width: 100rem;
-  padding: 2.5rem;
+.btn-in:disabled {
+  background-color: var(--white);
+  border: 1px solid #c1c7d0;
 }
 
-.about-header {
-  color: var(--color-headings);
-  font-size: 4rem;
-  margin-top: 5rem;
+.about {
+  display: flex;
+  flex-direction: column;
+  row-gap: 3em;
+  padding: 100px 24px;
+  background-color: var(--white);
+  border-top-right-radius: 5em;
 }
 
-.about-page p {
-  font-family: var(--font-family-style);
-  font-style: normal;
+.about h2 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 800;
+  color: #172b4d;
+  width: 60%;
+}
+
+.about p {
+  font-size: 16px;
   font-weight: 400;
-  font-size: 13px;
   line-height: 24px;
-  color: hsl(218, 31%, 30%);
-  flex: none;
-  order: 1;
-  margin: 12px 0;
+  color: #344563;
 }
 
-.about-button {
-  padding: 1.5rem 2.2rem;
-  margin-top: 3rem;
-  border-radius: 10px;
-  border: 1px solid #396df2;
-  color: #396df2;
-  font-size: 1.4rem;
+.user-tool-d {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.language-section {
-  margin-top: 3rem;
+.user-tool > div {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
 }
```

Comment:
- [ ] Please add the correct font size, font-weight, and margins to these heading text and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/167132179-7a1cb746-d98f-4016-a33b-5f39c9839f0f.png)  |![image](https://user-images.githubusercontent.com/68796232/167132162-70c98e9a-5f8c-4456-9a63-cd4f5944cedb.png) |